java-checks/src/test/files/checks/spring/SpringSecurityDebugModeCheck.java

```diff
@@ -0,0 +1,12 @@
+package files.checks.spring;
+
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+
+@EnableWebSecurity(debug = true) // Noncompliant [[sc=20;ec=32]] {{Deactivate Spring Security's debug mode.}}
+@EnableWebSecurity(unknown = true, debug = true) // Noncompliant [[sc=36;ec=48]]
+@EnableWebSecurity(debug = false)
+@EnableWebSecurity(debug)
+@EnableWebSecurity(true)
```

Comment:
Why is that case compliant given the rule ? it should be noncompliant as per : https://docs.spring.io/spring-security/site/docs/4.2.5.RELEASE/apidocs/org/springframework/security/config/annotation/web/configuration/EnableWebSecurity.html debug is the only attribute.

Reply:
Attribute name can be omitted only when it is called "value". Here, it is semantically not valid java code, because "EnableWebSecurity" does not have a "value" attribute. I added this case only for coverage reason.